tests/unit/archiver/test_manifest.py

```diff
@@ -34,9 +34,10 @@ def test_get_assay_process(self):
         ingest_api_mock.get_manifest_by_id = MagicMock(return_value={'fileFilesMap': ['f1']})
         ingest_api_mock.get_file_by_uuid = MagicMock()
         related_entity_map = {
-            'derivedByProcesses': iter('p1')
+            'derivedByProcesses': iter(['p1'])
```

Comment:
why make an iterator? I know it's not you but seems strange

Reply:
It is actually @aaclan-ebi , so I let her to answer ;-)

Reply:
This is the map of return values for ingest_api's get_related_entity mock. That function returns an iterator instead of the list of all processes related to an entity. If there are 1000 processes related to a biomaterial, the function will only return the iterator for the caller to retrieve 1000 processes one by one. This is more memory-efficient as it will not load all processes into memory.

Reply:
Good answer! Just seemed a bit strange when there's only a few values in the list

Reply:
ah yeah, coz it's just mocking it :)